### PR TITLE
Bump browser-run to latest, fixing electron security issue

### PR DESCRIPTION
**Pull request by eklem**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tape-run",
   "description": "Headless tape test runner",
-  "version": "7.0.0",
+  "version": "8.0.0",
```
Comment (juliangruber) [inline package.json:4]: this is done by my tooling, please revert that part

Comment (eklem) [reply package.json]: Done =)

**Comment by juliangruber** [comment]
thanks :) https://github.com/juliangruber/tape-run/releases/tag/v8.0.0

**Comment by eklem** [comment]
And thank you :-)

**Pull request merged**